irr/src/CMakeLists.txt

```diff
@@ -142,13 +142,10 @@ else()
 	option(ENABLE_OPENGL "Enable OpenGL" TRUE)
 endif()
 
-if(EMSCRIPTEN OR APPLE)
+if(USE_SDL2 OR EMSCRIPTEN OR APPLE)
```

Comment:
you add gles1 support for sdl and then forcibly disable it here?

Reply:
I see. I was just tripped up by the code change, because the end result of the logic is not any different.